### PR TITLE
Styles for posts and comments

### PR DESCRIPTION
**Pull request by a-nawrocka**
Some formatting done on index page + styles for posts and comments

### CODE REVIEW

**Review by ianmcnicholas — APPROVED** [review]
Great work!

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
-    <%= link_to "Comment", new_comment_path(post_id: post.id), class:"btn btn-outline-primary btn-sm" %>  <%# add query string parameter "post_id" %>
-  <% end %>
-<br>
-	<hr class="my-4"/>
-<% end %>
-
-	</p>
-
+    <% @posts.each do |post| %>
+      <div class="card post">
```
Comment (peter-james-allen) [inline app/views/posts/index.html.erb:4]: I love it!

**Review by kasey-purvor — APPROVED** [review]
I dont know if I can sign off on this ;)

**Review by peter-james-allen — COMMENTED** [review]
This is great 10/10, love those borders

**Review by anioji94 — APPROVED** [review]
I think this CSS is very cool.

**Pull request merged**